src/Login.tsx

```diff
@@ -1,9 +1,112 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router";
+// import axios from "axios";
 
-const Login = () => (
-	<div>
-		<h1>Login Page</h1>
-	</div>
-);
+type FormFieldsType = {
+	email: { value: string };
+	password: { value: string };
+} & EventTarget;
+
+const Login = () => {
+	const navigate = useNavigate();
+	const [error, setError] = useState<string>();
+	const [loginStatus, setLoginStatus] = useState(false);
+
+	const isValid = (e: React.FormEvent<HTMLFormElement>): boolean => {
+		const target = e.target as FormFieldsType;
+
+		const emailInput = target.email.value;
+		const passwordInput = target.password.value;
+		if (emailInput === "" || passwordInput === "") {
+			setError("some field are incomplete");
+		} else if (passwordInput.length < 8) {
+			setError("password must be more than 8 characters");
+		} else if (emailInput.includes("@") === false) {
+			setError("Email invalid");
+		} else {
+			return true;
+		}
+		return false;
+	};
+	const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+		e.preventDefault();
+
+		const target = e.target as FormFieldsType;
+		const email = target.email.value;
+		const password = target.password.value;
+
+		if (isValid(e)) {
+			const correctEmail = target.email.value;
+			const correctPassword = target.password.value;
+
+			if (correctEmail === email && correctPassword === password) {
+				setLoginStatus(true);
+				navigate("/profile");
+			} else {
+				setError("Error... Something went wrong");
+			}
+		}
+
+		// axios
+		// 	.post("http://localhost:8080/customers", {
+		// 	email,
+		// 	password
+		// 	})
+		// 	.then(response => {
+		// 	if (response.data == true) {
```

Comment:
I don't think you need to check response.data == true because I think you might get false negatives because I don't think response.data can == true? 
I think it would be better to just set the logged in boolean to true since you are already assuming that the .post call was successful and the back end has done it's validations

Reply:
Also, from the back end I will be sending you a "safe user" meaning it's the id and username etc NO PASSWORD. You will probably have to use that info from the response.data so you can have the id available on the front end.

So in addition to setting the loginStatus you will need to set a loggedInUser with a type of safe user

Reply:
this has been fixed